src/serverOperations/crudOperations.js

```diff
@@ -0,0 +1,32 @@
+import { fetchFromBase } from './fetchFromBase';
+
+export async function createRecord(base, data, sessionID) {
+    return await fetchFromBase(`${base}/add`, { ...data, sessionID }, 'post');
+}
+
+export async function getRecords(
+    base,
+    fromRecordID,
+    recordAmount,
+    params,
+    sessionID,
+) {
+    fromRecordID = fromRecordID > 0 ? fromRecordID : undefined;
```

Comment:
This is really unreadable to me. fromRecordID - from what? Please rename this variable to be more explanatory